sdk_extensions/logging_support/src/main/java/io/opentelemetry/logging/DefaultLogRecord.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.logging;
+
+import io.opentelemetry.common.AnyValue;
+import io.opentelemetry.common.AttributeValue;
+import io.opentelemetry.logging.api.LogRecord;
+import java.util.Map;
+
+public class DefaultLogRecord implements LogRecord {
```

Comment:
Any reason not to use AutoValue for this?

Reply:
The intent is to allow for flexibility in implementing these records- for example I would prefer to make an adapter for a Log4j record rather than copying each of the fields.

Reply:
We can make that change if we need to later, it's sort of an implementation detail. Since this seems to be a simple POJO for now, we may as well get sane equality / tostring for free while we're here